spec/inputs/s3_spec.rb

```diff
@@ -69,6 +69,21 @@
       expect(config.list_new_files).to eq([present_object.key])
     end
 
+    it 'should ignore file if the file match the prefix' do
```

Comment:
Just personal test, but I find using should in descriptions a bit strange (http://betterspecs.org/#should)